k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -84,6 +84,10 @@ spec:
             configMapKeyRef:
               name: bdb-config
               key: bigchaindb-log-level
+        - name: BIGCHAINDB_SERVER_WORKERS
+          value: "1"
+        - name: BIGCHAINDB_SERVER_THREADS
```

Comment:
`BIGCHAINDB_SERVER_WORKERS` is [documented in the BigchainDB Server docs](https://docs.bigchaindb.com/projects/server/en/master/server-reference/configuration.html#server-bind-server-loglevel-server-workers), but `BIGCHAINDB_SERVER_THREADS` isn't. Should we should add docs for the latter?

Maybe not. I get the feeling that this business of setting the number of threads to 1 is a temporary workaround and one wouldn't normally ever do it.

Reply:
Yes, I agree. We won't need these parameters in the future as well, hopefully.